### DISCUSSION TITLE
多表连接的Where条件中的Expression表达式问题

### DISCUSSION THREAD

**Discussion by AshWang-whm** [dotnetcore/FreeSql#1753] [Q&A]
![image](https://github.com/dotnetcore/FreeSql/assets/31700716/bfdc2930-618d-42cf-ad27-f4981b8cf539)
我这边生成一个两表连接的Expression，如果只输入表1(TrainingRecord)的条件可以正常运行， 输入表2(TrainingGroup)的条件，比如GroupName字段，会提示 TrainingRecord 没有 GroupName字段，这里应该是类似  exp.invoke(p1,p1) 处理了吗， 正常应该是exp.invoke(p1,p2) ?

**Comment by 2881099** [comment]
表达式树的拼接，建议自己分析一下
